### PR TITLE
COMMERCE-10454

### PR DESCRIPTION
**Pull request by Brunno0liveira**

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:stop

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase [hunk 1]
```diff
@@ -14,6 +14,42 @@ definition {
...
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
+
+			CommerceAcceleratorsInitializer.addPortlet(portletName = "Product Publisher");
+
+			Portlet.gotoPortletOptions(
+				portletName = "Product Publisher",
+				portletOption = "Configuration");
+		}
+
+		task ("And the Product Entries is not shown") {
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase:30]: Can you fix it into "And the Product Entries are not shown"?

Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase [hunk 2]
```diff
@@ -14,6 +14,42 @@ definition {
...
+				locator1 = "CommerceEntry#CONFIGURATION_PANEL_EXPANDED");
+		}
+
+		task ("When the Manual option is selected") {
+			Click(
+				locator1 = "Radio#ANY",
+				radioLabel = "Manual");
+		}
+
+		task ("Then the Product Entries is automatically shown") {
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase:46]: Can you fix it into "Then the Product Entries are automatically shown"?

Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase [hunk 3]
```diff
@@ -14,6 +14,42 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-10454."
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase:17]: Can you add something more to the description that explains quickly what this test is doing?

Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase [hunk 4]
```diff
@@ -14,6 +14,42 @@ definition {
...
+	@priority = 3
+	test CanViewProductSourceUpdateWhenChangedFromDynamicToManual {
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase:19]: Can you change the name in "CanAssertProductSourceUpdated"?

Thanks!

**Review by pietrobocale — CHANGES_REQUESTED** [review]

**Comment by pietrobocale** [comment]
Review completed. @Brunno0liveira, take a look at the comments. 

Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
Re-sent at #3432

**Comment by lorenzolapescara** [comment]
ci:close

**Pull request closed without merging**